api/controllers/delegates.js

```diff
@@ -33,6 +33,7 @@ var modules;
  */
 function DelegatesController(scope) {
 	modules = scope.modules;
+	constants = scope.config.constants;
```

Comment:
I don't really like that, can we be consistent? Why not use `global.constants` here?